### ISSUE TITLE
VSCode overwriting my file with old content on save

### ISSUE THREAD

**Issue by jessep** [reasonml-editor/vscode-reasonml#149]
I believe this is happening in the process of formatting on save. It is appending the old contents to the current contents. bmcmahen on discord is having the same issue.

**Comment by osenvosem** [comment]
The same issue here, only restarting VS Code helps.

**Comment by blogscot** [comment]
I found the problem is caused when `editor.formatOnSave` is enabled in the configuration settings, `settings.json`.

Also, with `reason.formatOnSave` enabled, as below, `refmt` does not appear to run in the editor.

```javascript
{
  "reason.formatOnSave": true,
  "editor.formatOnSave": false,
}
```

**Comment by freebroccolo** [comment]
This should be fixed in version `1.0.23`.
